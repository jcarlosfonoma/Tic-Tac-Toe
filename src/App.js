import "./App.css";
import GameContainer from "./components/GameContainer";
import ScoreBoard from "./components/ScoreBoard";
import styled from "styled-components";

const Wrapper = styled.div`
  min-width: 500px;
  min-height: 500px;
`;

function App() {
  return (
    <Wrapper>
      <ScoreBoard />
      <GameContainer />
    </Wrapper>
  );
}

export default App;
