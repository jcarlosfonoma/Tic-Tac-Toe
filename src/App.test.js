import { render, cleanup, screen } from "@testing-library/react";
import App from "./App";

afterEach(cleanup);

it("should take a snapshot", () => {
  const { asFragment } = render(<App />);

  expect(asFragment(<App />)).toMatchSnapshot();
});