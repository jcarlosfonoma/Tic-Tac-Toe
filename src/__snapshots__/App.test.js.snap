// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-furwcr fPbZyz"
  >
    <div
      class="sc-gKclnd fWTgoL"
    >
      <span
        class="sc-iCfMLu iZPcPr"
        data-testid="x-score-span"
      >
        X Wins: 0
      </span>
      <span
        class="sc-iCfMLu iZPcPr"
        data-testid="o-score-span"
      >
        O Wins: 0
      </span>
      <span
        class="sc-iCfMLu iZPcPr"
        data-testid="draws-span"
      >
        Draws: 0
      </span>
    </div>
    <div
      class="game"
    >
      <div
        class="game-board"
      >
        <div>
          <div
            class="board-row"
          >
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
          </div>
          <div
            class="board-row"
          >
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
          </div>
          <div
            class="board-row"
          >
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
            <button
              class="sc-bdvvtL fPtRnW"
              data-testid="square-btn"
            />
          </div>
        </div>
      </div>
      <div
        class="sc-gsDKAQ gKFSjR"
      >
        <div
          class="sc-dkPtRN hDBrjf"
          data-testid="status-div"
        >
          Next player X
        </div>
        <ol
          class="sc-hKwDye fqVKqO"
        >
          <li
            class="sc-eCImPb jiTVab"
          >
            <button
              class="sc-jRQBWg bShOYO"
            >
              Go to game start
            </button>
          </li>
        </ol>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
